src/components/InputRange/index.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import { Container, Wrapper } from './styles';
+
+export const InputRange = () => {
+  const [rangeValue, setRangeValue] = useState(2);
```

Comment:
vale esse valor ser recebido pela prop

```suggestion
export const InputRange = ({ initialRange }) => {
  const [rangeValue, setRangeValue] = useState(initialRange);
```
Isso se chama Estado Derivado (Derived State)

cabe a pesquisa pra entender melhor.